packages/celotool/src/e2e-tests/transfer_tests.ts

```diff
@@ -83,6 +83,19 @@ class InflationManager {
   }
 }
 
+const setGasParameters = async (
```

Comment:
nit: consider renaming to `setIntrinsicGasForAlternativeGasCurrency` or `setIntrinsicGas`